app/javascript/react/store/fixedStreamSelectors.ts

```diff
@@ -137,11 +137,14 @@ const selectFixedStreamShortInfo = createSelector(
   (fixedStreamData, lastDailyAverage): FixedStreamShortInfo => {
     const { value: lastMeasurementValue, date } = lastDailyAverage || {};
     const lastMeasurementDateLabel = moment(date).format("MMM D");
+    const formattedDate = moment(fixedStreamData.stream.lastUpdate).local().format("HH:mm, MMM D YYYY");
+    const lastUpdate = formattedDate + " (local time)";
```

Comment:
I'd return only the formatted date and cover the local time addition with translations, there is a way to pass a variable content with translation key, where you can define the final structure of your label. 
We have such cases in the app already, lmk if it's not clear.

Reply:
Done